clients/samples/CMakeLists.txt

```diff
@@ -63,7 +61,7 @@ if( NOT TARGET hipblas )
 endif( )
 
 list (APPEND hipblas-example-executables hipblas-example-sscal hipblas-example-scal-ex-v2 hipblas-example-strmm hipblas-example-sgemm hipblas-example-sgemm-strided-batched hipblas-example-gemmEx_v2 hipblas-example-hip-complex-her2 hipblas-example-hgemm-half hipblas-example-c ${sample_list_fortran} )
-if( CMAKE_CXX_COMPILER MATCHES ".*/hipcc$" )
+if( CMAKE_CXX_COMPILER MATCHES ".*/hipcc$" OR CMAKE_CXX_COMPILER MATCHES ".*/amdclang\\+\\+$")
```

Comment:
Are you using hipcc on windows?  Otherwise you can try:
CMAKE_CXX_COMPILER_ID MATCHES "Clang" for more general conditional